modules/nw-metallb-operator-limit-speaker-to-nodes.adoc

```diff
@@ -21,7 +21,7 @@ apiVersion: metallb.io/v1beta1
 kind: MetalLB
 metadata:
   name: metallb
-  namespace: metallb-system
+  namespace: openshift-operators
```

Comment:
openshift-operators still present. Can you do a search to see if we still have leftovers?